.gitignore

```diff
@@ -1,3 +1,4 @@
+node_modules/
 # Build and Release Folders
```

Comment:
- [ ] Great job including the node_modules/ in the `.gitignore`  file but it seems you missed including the `test.md` file as well.
It's so important to `gitignore` all the unrelated files to the code.
Please about [gitignoring committed files ](https://docs.github.com/en/github/getting-started-with-github/getting-started-with-git/ignoring-files) to do so.